src/Codeception/Module/Symfony2.php

```diff
@@ -93,17 +99,18 @@ class Symfony2 extends Framework implements DoctrineProvider, PartedModule
     public $kernel;
 
     /**
-     * @var \Symfony\Component\DependencyInjection\ContainerInterface
+     * @var ContainerInterface
      */
-    public $container;
+    private $container;
```

Comment:
We can make `$container` a read-only property and retrieve it with magic method `__get($property)`